Runner/Program.cs

```diff
@@ -38,74 +38,72 @@ public static void Main(string[] args)
-                client.Init().Wait();
-
-                void scanner(IEnumerable<string> repositories, Options options)
+                var start = DateTime.UtcNow;
+                var results = repositories.Select(repo =>
                 {
-                    var start = DateTime.UtcNow;
-                    var results = repositories.Select(repo =>
-                    {
-                        // This sleeps used to avoid hitting GitHub API limits
-                        Thread.Sleep(TimeSpan.FromSeconds(3));
-                        return client.ValidateRepository(githubConfig.Organization, repo, options.IgnoreRepositoryRules).ConfigureAwait(false).GetAwaiter().GetResult();
-                    }).ToArray();
+                    // This sleeps used to avoid hitting GitHub API limits
+                    Thread.Sleep(TimeSpan.FromSeconds(3));
+                    return client.ValidateRepository(githubConfig.Organization, repo, options.IgnoreRepositoryRules).ConfigureAwait(false).GetAwaiter().GetResult();
+                }).ToArray();
 
-                    ReportToConsole(logger, results);
+                ReportToConsole(logger, results);
 
-                    if (options.AutoFix)
-                    {
-                        PerformAutofixes(ghClient, results);
-                    }
-                    if (!string.IsNullOrWhiteSpace(options.CsvFile))
-                    {
-                        ReportToCsv(di.GetService<ILogger<CsvReporter>>(), options.CsvFile, results);
-                    }
-                    if (options.ReportToGithub)
-                    {
-                        ReportToGitHub(ghClient, GitHubReporterConfig, di.GetService<ILogger<GitHubReporter>>(), results).Wait();
-                    }
-                    if (options.ReportToSlack)
-                    {
-                        var slackSection = config.GetSection("Slack");
-                        if (slackSection.Exists())
-                        {
-                            var slackConfig = new SlackConfiguration();
-                            slackSection.Bind(slackConfig);
-                            ReportToSlack(slackConfig, logger, results).Wait();
-                        }
-                    }
-                    logger.LogInformation("Duration {duration}", (DateTime.UtcNow - start).TotalSeconds);
+                if (options.AutoFix)
+                {
+                    PerformAutofixes(ghClient, results);
                 }
-
-                Parser.Default.ParseArguments<ScanSelectedOptions, ScanAllOptions, DebugTestOptions>(args)
-                .WithParsed<ScanSelectedOptions>(options =>
+                if (!string.IsNullOrWhiteSpace(options.CsvFile))
                 {
-                    scanner(options.Repositories, options);
-                })
-                .WithParsed<ScanAllOptions>(options =>
+                    ReportToCsv(di.GetService<ILogger<CsvReporter>>(), options.CsvFile, results);
```

Comment:
Why theres CSV reports 😄

Reply:
Ah, that was just for my personal use, I was making a trend report (are issues actually fixed etc) :D